tests/page-templates/18-usertiming/04-usertiming-second-beacon.html

```diff
@@ -0,0 +1,30 @@
+<%= header %>
+<%= boomerangSnippet %>
+<script src="04-usertiming-second-beacon.js" type="text/javascript"></script>
+<script src="../../vendor/usertiming-compression/src/usertiming-decompression.js" type="text/javascript"></script>
+<script>
+
+  if (BOOMR_test.isUserTimingSupported()) {
+    window.performance.mark("pre-load-mark-start");
+    window.performance.mark("pre-load-mark-end");
+    window.performance.measure("pre-load-measure", "pre-load-mark-start", "pre-load-mark-end");
+  }
+
+  function sendPostLoadMark() {
+    if (BOOMR_test.isUserTimingSupported()) {
+      window.performance.mark("post-load-mark-start");
+      window.performance.mark("post-load-mark-end");
+      window.performance.measure("post-load-measure", "post-load-mark-start", "post-load-mark-end");
+      BOOMR.requestStart('test').loaded();
+    }
+  }
+
+  BOOMR_test.init({
+    testAfterOnBeacon: true,
```

Comment:
Change to `testAfterOnBeacon: 2,`. This will make the test to wait for 2 beacons before running checks otherwise it will be timing sensitive.

Reply:
Cool! I wasn't sure what that variable did.